packages/terminal/src/browser/terminal-preferences.ts

```diff
@@ -39,6 +39,7 @@ export const TerminalConfigSchema: PreferenceSchema = {
         'terminal.integrated.fontSize': {
             type: 'number',
             description: 'Controls the font size in pixels of the terminal.',
+            minimum: 9,
```

Comment:
I wonder, should we align with VSCode's minimum font size?

https://github.com/Microsoft/vscode/blob/fc277b039ba07ed734ee15b37869bd86f8ae1403/src/vs/workbench/contrib/terminal/browser/terminalConfigHelper.ts#L18

Reply:
Imo, at size 6, the terminal is almost unreadable already 
![image](https://i.gyazo.com/4761ec6a338eb74bbddc0016c5a793d3.png)
I'd go for at least 8,
![image8](https://i.gyazo.com/65a2e45cde1acee7304cc2a6400de611.png)

Reply:
We are going to be compatible with setting.json file of VS Code, any value which is allowed there should be allowed in Theia.